src/shared/utilities/vsCodeUtils.ts

```diff
@@ -0,0 +1,169 @@
+/*!
+ * Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+'use strict'
+import * as vscode from 'vscode'
+import * as nls from 'vscode-nls'
+import { EnvironmentVariables } from '../environmentVariables'
+export const localize = nls.loadMessageBundle()
+
+// TODO: Why are messages from bundle never displayed?
+const env = process.env as EnvironmentVariables
+if (!!env.VSCODE_NLS_CONFIG) {
+    nls.config(JSON.parse(env.VSCODE_NLS_CONFIG) as nls.Options)()
+} else {
+    nls.config()()
+}
+
+export enum OutputChannelName {
+    ToolKit = 'ToolKit',
+    Lambda = 'Lambda'
+}
+const outputChannels: {[channelName: string]: vscode.OutputChannel} = {
+    [OutputChannelName.ToolKit.toString()]: vscode.window.createOutputChannel(
+        localize('AWS.channel.aws.toolkit', 'AWS Toolkit')
+    ),
+    [OutputChannelName.Lambda.toString()]: vscode.window.createOutputChannel('AWS Lambda')
+}
+
+export const getOutputChannel = (name: OutputChannelName) => outputChannels[name.toString()]
+
+// ------------------- Placeholder for upcoming logger -------------------
+// TODO: Remove this code and replace usage with upcoming src/shared/logger
+const loggerChannel = vscode.window.createOutputChannel('AWS Toolkit Logs')
+export type ErrorOrString = string | Error | number
+
+export interface Logger {
+    debug(...message: ErrorOrString[]): void
+    info(...message: ErrorOrString[]): void
+    warn(...message: ErrorOrString[]): void
+    error(...message: ErrorOrString[]): void
+}
+
+const doLog = (...message: ErrorOrString[]): void => {
+    // Ignore level for now. Log to both comnsole and output channel
+    console.log(...message)
+    loggerChannel.show()
+    loggerChannel.appendLine(message.join(' '))
+}
+
+export const getLogger = (): Logger => {
+
+    return {
+        debug: doLog,
+        info: doLog,
+        warn: doLog,
+        error: doLog
+    }
+}
+
+// ---------------------- End placeholder for upcoming logger ----------------
```

Comment:
The logger PR is merged now, I'll review this section after you merge `develop`.

Reply:
Done.
✔️